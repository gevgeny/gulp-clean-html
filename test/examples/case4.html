<div>
    <div to-remove1="foo1">
        node
    </div>
    <div to-remove1="bar1">
        node
    </div>
    <div to-remove1="baz1">
        node
    </div>
</div>
<div to-remove2="foo2">
    node
</div>
<div to-remove2="bar2">
    node
</div>
<div to-remove2="baz2">
    node
</div>
