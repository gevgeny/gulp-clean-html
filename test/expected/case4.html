<div>
    
    
    <div to-remove1="baz1">
        node
    </div>
</div>


<div to-remove2="baz2">
    node
</div>
